zmon-controller-app/src/main/java/org/zalando/zmon/config/FeedbackMetadataProperties.java

```diff
@@ -6,13 +6,13 @@
 public class FeedbackMetadataProperties {
     private String url;
 
-    public boolean feedbackEnabled = false;
+    private boolean enabled;
 
     public String getUrl() { return this.url; }
 
     public void setUrl(String url) { this.url = url; }
 
-    public boolean getFeedbackEnabled() { return feedbackEnabled; }
+    public boolean getFeedbackEnabled() { return enabled; }
```

Comment:
name of the field and getter/setter should match.

Reply:
thanks, I'll make the changes now